app/(root)/case-studies/page.tsx

```diff
@@ -1,4 +1,4 @@
-import CaseStudies from '@/components/caseStudies/CaseStudies';
+import CaseStudies from '@/components/case-studies/CaseStudies';
 
 const Page = () => {
```

Comment:
Update name as mentioned previously. If this is only responsible for returning "Case Studies," maybe consider using `CaseStudies` instead of `Page`.